### PR TITLE
Pinephone support

### PR DESCRIPTION
**Pull request by Icenowy**
Depends on my PR at manifests that adds 8723cs.

### CODE REVIEW

File: pinephone/device.mk [hunk 1]
```diff
@@ -5,7 +5,7 @@
 $(call inherit-product, $(SRC_TARGET_DIR)/product/core_64_bit.mk)
 $(call inherit-product, device/glodroid/common/device-common.mk)
 
-PRODUCT_PROPERTY_OVERRIDES += ro.sf.lcd_density=269
+PRODUCT_PROPERTY_OVERRIDES += qemu.sf.lcd_density=269
```
Comment (rsglobal) [inline pinephone/device.mk:8]: Are you sure about this? 
Try to drop this line.

Comment (Icenowy) [reply pinephone/device.mk:8]: I'm sure that ro. will makes the DPI set to still 160, which leads to very small UI components.

Comment (Icenowy) [reply pinephone/device.mk:8]: Maybe we need some better way to management the setting of ro.sf.lcd_density .

Comment (rsglobal) [reply pinephone/device.mk:8]: Ah. I see:

```
./frameworks/native/services/surfaceflinger/SurfaceFlinger.cpp:876:                // if "qemu.sf.lcd_density" is specified, it overrides everything
```

Lets keep it as is for now.

**Comment by rsglobal** [comment]
Looks good. Merging.

**Pull request merged**